hadoop-hdfs-project/hadoop-hdfs-rbf/pom.xml

```diff
@@ -177,6 +177,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
```

Comment:
nit: make the indentation consistent with the others

Reply:
Git dulpicated somehow, removed extra one.